src/wp-includes/class-wp-block-patterns-registry.php

```diff
@@ -164,12 +164,16 @@ public function unregister( $pattern_name ) {
 	 */
 	private function prepare_content( $pattern, $hooked_blocks ) {
 		$content = $pattern['content'];
+
+		$before_block_visitor = '_inject_theme_attribute_in_template_part_block';
+		$after_block_visitor  = null;
+		$hooked_blocks        = get_hooked_blocks();
 		if ( ! empty( $hooked_blocks ) || has_filter( 'hooked_block_types' ) ) {
-			$blocks               = parse_blocks( $content );
-			$before_block_visitor = make_before_block_visitor( $hooked_blocks, $pattern );
-			$after_block_visitor  = make_after_block_visitor( $hooked_blocks, $pattern );
-			$content              = traverse_and_serialize_blocks( $blocks, $before_block_visitor, $after_block_visitor );
+			$before_block_visitor = make_before_block_visitor( $hooked_blocks, $template );
+			$after_block_visitor  = make_after_block_visitor( $hooked_blocks, $template );
```

Comment:
```suggestion
			$before_block_visitor = make_before_block_visitor( $hooked_blocks, $pattern );
			$after_block_visitor  = make_after_block_visitor( $hooked_blocks, $pattern );
```

This should fix the failing unit tests.

Reply:
Going to commit to get unit tests to pass.